src/common/transformations/src/transformations/common_optimizations/compress_float_constants.cpp

```diff
@@ -96,7 +96,8 @@ class DetectCompressedWeights : public MatcherPass {
                                                                                            ov::element::u8,
                                                                                            ov::element::nf4,
                                                                                            ov::element::f8e4m3,
-                                                                                           ov::element::f8e5m2}));
+                                                                                           ov::element::f8e5m2,
+                                                                                           ov::element::f8e8m0}));
```

Comment:
The transformation can be skipped for now.
Can be expanded later if required.
@itikhono, could you confirm?

Reply:
Done